aware-core/src/main/java/com/aware/Aware.java

```diff
@@ -304,6 +305,15 @@ protected Boolean doInBackground(Void... params) {
             Hashtable<String, String> device_ping = new Hashtable<>();
             device_ping.put(Aware_Preferences.DEVICE_ID, Aware.getSetting(awareContext, Aware_Preferences.DEVICE_ID));
             device_ping.put("ping", String.valueOf(System.currentTimeMillis()));
+            device_ping.put("platform", "android");
+            try {
+                PackageInfo package_info = awareContext.getPackageManager().getPackageInfo(awareContext.getPackageName(), 0);
```

Comment:
Using context to get the package name will mean that this will be different per application, if AWARE is a library. Also, this should only happen for the AWARE client, not the plugins. So do a check for "com.aware.phone" package.